velox/type/Variant.cpp

```diff
@@ -555,7 +555,14 @@ uint64_t variant::hash() const {
       return folly::Hash{}(
           timestampValue.getSeconds(), timestampValue.getNanos());
     }
-
+    case TypeKind::SHORT_DECIMAL: {
```

Comment:
Can you verify if we still need these? Thanks.

Reply:
Responded above.